README.md

```diff
@@ -1,2 +1,87 @@
-# transac_center
-This application allows you to have an eye on your expenses. Track your expenses closely and pay attention to how you spend your income.
+# Transac_Center
+
+# Live demo
+[Transac_center](https://pacific-taiga-56560.herokuapp.com)
+
+# Preview
+![](./preview_1.png)
+![](./preview_2.png)
+
+## Built With
+
+- Ruby on Rails
+- PostgreSQL
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- [Ruby](https://www.ruby-lang.org/en/)
+- [Rails](https://gorails.com/)
+
+### Setup
+
+- Make sure you have Ruby on Rails set up properly on your computer
+- Clone or download this repo on your machine
+- Enter project directory
+
+### Development Database
+
+```sh
+# Create user
+sudo -u postgres createuser -e ced00 -s
+# Load the schema
+rails db:schema:load
+# Create the database
+rake db:create
+# Run database Migration
+rails db:migrate
+```
+
+### Install
+
+```sh
+bundle install
+```
+
+### Run server
+
+```sh
+rails s
+```
+
+### Test
+
+```sh
+rspec
+```
+
+## Authors
```

Comment:
- [ ] OPTIONAL
Kindly change the header title _Authors_ to _Author_, as this project is done solely. This will make your ReadMe appear more professional.

Reply:
Thank you. I am going to update my Readme file.